### ISSUE TITLE
API Proposal: Broadcasts with Layout

### ISSUE THREAD

**Issue by aoberoi** [opentok/Opentok-Python-SDK#90]
This is a request for comments regarding the API for upcoming platform functionality. The platform functionality is not generally available at the moment and there is no guarantee it will be released in any form.

this is related to #83 
### Create a Broadcast

``` python
# each returns a `Broadcast` instance

# simplest case
broadcast = opentok.create_broadcast(session_id)

# using predefined layout
# `Layouts` is a subclass of Enum, the values are dicts that mimic the JSON payload in the REST API
broadcast = opentok.create_broadcast(session_id, layout=Layouts.PIP)

# using custom layout
# `Layouts.create_custom()` is a class method that returns a dict that mimics the JSON payload in the REST API, will use https://pypi.python.org/pypi/aenum to extend_enum
broadcast = opentok.create_broadcast(session_id, layout=Layouts.create_custom(stylesheet="{css string}"))
```
### Stop a Broadcast

``` python
# each returns a `Broadcast` instance

# when you don't have an instance of Broadcast
broadcast = opentok.stop_broadcast(broadcast_id)

broadcast.stop()
```
### Get a Broadcast

``` python
# returns a `Broadcast` instance
broadcast = opentok.get_broadcast(broadcast_id)

# `Broadcast` has the following properties
broadcast.id; // string
broadcast.sessionId; // string
broadcast.createdAt; // long
broadcast.updatedAt; // long
broadcast.partnerId; // string
broadcast.urls; // dict which contains a key `"HLS"` with a value of a URL
```
### Get a Broadcast Layout

Question: should we extend the Layouts enum and enforce that this return value is part of it?

``` python
# each returns a dict with keys `type` and optionally `stylesheet`

# when you don't have an instance of Broadcast
layout = opentok.get_broadcast_layout(broadcast_id)

layout = broadcast.get_layout()
```
### Update a Broadcast Layout

``` python
# each returns None

# when you don't have an instance of Broadcast, 
opentok.update_broadcast_layout(broadcst_id, layout)

broadcast.update_layout(layout)
```
### Update a Stream's Layout Class List

Question: should we have an API for updating multiple streams?
Question: should these return a representation of the Stream properties?

``` python
# each returns None

# when you don't have a Session instance
opentok.update_stream(session_id, stream_id, layout_class_list=["class1"])

session.update_stream(stream_id, layout_class_list=["class1", "class2"])
```

**Comment by aoberoi** [comment]
we plan to begin an implementation next week. please comment at your earliest convenience.

**Comment by msach22** [comment]
This functionality has been released with `v2.10.0`
